onecodex/api.py

```diff
@@ -43,6 +43,9 @@ def __init__(
             if base_url != "https://app.onecodex.com":
                 warnings.warn("Using base API URL: {}".format(base_url))
 
+        if schema_path != "/api/v1/schema":
```

Comment:
Add `**kwargs` and check if `experimental` in kwargs, warn about not being supported, and switch schema_path to experimental